### PR TITLE
SAMZA-1290: Fix javadoc for ApplicationRuner

### PR DESCRIPTION
**Pull request by jmakes**

### CODE REVIEW

**Comment by jmakes** [comment]
@prateekm

File: samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 
 /**
- * A physical execution engine to deploy the config and jobs to run the {@link org.apache.samza.operators.StreamGraph}
+ * The primary means of interacting with the {@link org.apache.samza.application.StreamApplication} at runtime.
```
Comment (prateekm) [inline samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java]: s/interacting with/running?

Comment (jmakes) [inline samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java]: I was going to say that but it also kills and gets status. Do we consider those be under the umbrella of "running"?

Comment (prateekm) [inline samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java]: Good point. Maybe managing the execution? Interacting is fine too.

**Pull request closed without merging**